sklearn/cluster/tests/test_k_means.py

```diff
@@ -966,11 +1039,91 @@ def test_minibatch_kmeans_partial_fit_int_data():
     assert km.cluster_centers_.dtype.kind == "f"
 
 
-def test_result_of_kmeans_equal_in_diff_n_jobs():
+def test_result_of_kmeans_equal_in_diff_n_threads():
     # PR 9288
```

Comment:
maybe remove since it's not relevant anymoer

Reply:
I keep the test because I think it's good to test that the results are the same in sequential mode and non sequential mode. I updated the comment to reflect that.